providers/resource/bin/v4l2_ioctl_resource.py

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/python3
+
+from checkbox_support.parsers.v4l2_compliance import TEST_NAME_TO_IOCTL_MAP
+import subprocess as sp
+
+
+def main():
+    udev_out = sp.check_output(
+        "udev_resource.py -f CAPTURE", universal_newlines=True, shell=True
```

Comment:
I'm not really sure if this is the correct way to access udev outputs inside a resource job since it uses `shell=True`. What would be the better way to do this? :thinking:

Reply:
Fixed, directly use Udevdm from checkbox support instead